thewalrus/quantum.py

```diff
@@ -713,3 +716,68 @@ def is_classical_cov(cov, hbar=2, atol=1e-08):
         if np.all(vals >= 0):
             return True
     return False
+
+
+def gen_single_mode_dist(s, nmax=50, N=1):
+    """Generate the photon number distribution of N identical single mode squeezed states with squeezing parameter s up to cutoff nmax
+    Args:
+        s (float): squeezing paramer
+        nmax (int): Fock cutoff

```

Comment:
Should we make this 'cutoff' to be consistent with the other functions in the quantum module?

Reply:
No. This is, at least conceptually, not the same cutoff.